package.json

```diff
@@ -9,39 +9,70 @@
   "license": "MIT",
   "homepage": "https://github.com/jsonapi-suite/jsorm",
   "scripts": {
-    "clean": "shx rm -rf _bundles lib lib-esm",
-    "build": "shx cp es-shims/* src/util/ && npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack && shx cp node-shims/* src/util/ ",
-    "test": "mocha -r test/tshook.js test/**/*.ts || true"
+    "f:clean": "shx rm -rf _bundles lib lib-esm",
```

Comment:
What's the `f` stand for?

Reply:
Random prefix (can't remember the original reason for the choice of `f`) to save out the old versions of these while I still rework everything.